frontend/src/services/auth/auth-service.js

```diff
@@ -0,0 +1,69 @@
+// Services for auth module to access
+import axios from 'axios';
+import authHeader from './auth-header';
+
+const API_URL = 'http://127.0.0.1/api/auth/';
```

Comment:
We need to make a env variable in the frontend for the backend api URL eventually